content/patterns/feed/examples/feed.html

```diff
@@ -39,11 +39,29 @@ <h2>About This Example</h2>
         <p>Unlike other examples in the WAI-ARIA Authoring Practices, the example experience has its own page separate from this documentation page.</p>
       </section>
 
+      <section>
+        <h2>Example Options</h2>
+
+        <fieldset class="feed-options">
+          <legend>Feed Options</legend>
+
+          <select id="delay-time-select" name="delay_time" aria-describedby="feed-controls-desc">
+            <option value="200">200 ms</option>
+            <option value="400">400 ms</option>
+          </select>
+          <label for="delay-time-select"> Loading delay</label>
+
+          <p id="feed-controls-desc">This option determines the article loading delay time in this feed example.</p>
+
+        </fieldset>
+      </section>
+
       <section>
         <h2 id="ex_label">Example</h2>
         <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
         <div id="ex1">
-          <p>The example feed experience is presented on a separate <a href="feed-display.html">feed display page.</a></p>
+          <p>The example feed experience below is presented in an iframe in order not to obstruct from the rest of the content of the page. </p>
+          <iframe id="feed_frame" src="./feed-display.html" height="500" title="Feed example" style="width: 100%"></iframe>
```

Comment:
Hi @jugglinmike, the deploy preview will only work if I change the following:
```suggestion
          <iframe id="feed_frame" src="../feed-display.html" height="500" title="Feed example" style="width: 100%"></iframe>
```

but keeping it as `src="./feed-display.html"` gets the tests to correctly run and has it successfully work when viewing locally.

Do you have any ideas on how to go about fixing this?

Reply:
Hey there, Ariella! Although we were suspicious of the automated testing infrastructure during today's call, this appears to actually be a snag in the build process. I've diagnosed the problem and [proposed a solution](https://github.com/w3c/wai-aria-practices/pull/262); if that's accepted, then the preview should start to work using `./feed-display.html` as you expected (though you'll likely need to push another commit to trigger the system to regenerate).

Reply:
Thanks @jugglinmike!

Reply:
@ariellalgilmore @jugglinmike @mcking65 

What is the correct example for feed example link for the reivew? 
https://deploy-preview-248--aria-practices.netlify.app/aria/apg/patterns/feed/examples/feed-display does not show iframe feed example. The file path is aria/apg/patterns/feed/examples/feed-display

https://deploy-preview-248--aria-practices.netlify.app/aria/apg/patterns/feed/examples/feed/ still show page not fouud error.

@mcking65 The code pen button would for work for this example.